### PR TITLE
WIP: Review project README + change for F4PGA

### PR DESCRIPTION
**Pull request by mgielda**
Reviewed the README and while I was at it, also switched to the new name, so this will have to wait until the rename is completed + will require changes in the code (as symbiflow is used in make env etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,35 +2,35 @@
...
 - nextpnr-xilinx
 - yosys (+ yosys-plugins)
 - prjxray
 
-Prior to setting up the conda environment, the symbiflow and quicklogic data files need to be installed through the following commands:
+Prior to setting up the conda environment, the F4PGA and QuickLogic data files need to be installed through the following commands:
 
 ```bash
-make install_symbiflow
+make install_f4pga
```
Comment (acomodi) [inline README.md:18]: This change should be reflected also in the Makefile to have them in sync

**Review by acomodi — CHANGES_REQUESTED** [review]
LGTM, apart from some changes to be added to the Makefile as well (see comment below)

**Comment by mithro** [comment]
Make sure the symbiflow command names are kept for backward compatibility...

**Comment by umarcor** [comment]
Superseeded by #454.

**Pull request closed without merging**